pkg/bitbucket/webhooks_test.go

```diff
@@ -0,0 +1,37 @@
+package bitbucket
+
+import (
+	"testing"
+
+	"github.com/opendevstack/pipeline/test/testserver"
+)
+
+func TestWebhookCreate(t *testing.T) {
+	srv, cleanup := testserver.NewTestServer(t)
+	defer cleanup()
+	bitbucketClient := testClient(srv.Server.URL)
+
+	srv.EnqueueResponse(
+		t, "/rest/api/1.0/projects/PRJ/repos/my-repo/webhooks",
+		201, "bitbucket/webhook-create.json",
+	)
+
+	w, err := bitbucketClient.WebhookCreate(
+		"PRJ", "my-repo",
+		WebhookCreatePayload{
+			Name:   "Webhook Name",
+			Events: []string{"repo:refs_changed", "repo:modified"},
```

Comment:
Are these webhook events enough? I realized in our current ODS installation we also have:

![image](https://user-images.githubusercontent.com/15997951/124231684-e867ab80-db10-11eb-8376-2a49f68eb164.png)

Reply:
It's just a test. I always use the examples from the documentation so keeping as just one event.